src/index.js

```diff
@@ -8,20 +8,20 @@ import type { Options } from './types'
 /**
  * Returns a webpack block that splits vendor javascript bundle.
  */
-const splitVendor = (options: string | Options): Function => {
-  const {
-    name = 'vendor',
-    exclude,
-  } = typeof options === 'object' ? options : { name: options, exclude: undefined }
+const splitVendor = (nameOrOptions: string | Options): Function => {
+  const options: Options = typeof nameOrOptions === 'object' ? nameOrOptions : { name: nameOrOptions }
+  if (options.name == null) options.name = 'vendor'
+  /* istanbul ignore next */
+  if (options.test == null) options.test = /\.js$/
```

Comment:
Don't mutate arguments and let's not ignore coverage. It's there for a reason, I can write tests later if you can't.